services/webdav/pkg/command/server.go

```diff
@@ -50,19 +50,19 @@ func Server(cfg *config.Config) *cli.Command {
 					}
 					return context.WithCancel(cfg.Context)
 				}()
-				metrics = metrics.New()
+				m = metrics.New()
 			)
 
 			defer cancel()
 
-			metrics.BuildInfo.WithLabelValues(version.GetString()).Set(1)
+			m.BuildInfo.WithLabelValues(version.GetString()).Set(1)
 
 			{
 				server, err := http.Server(
 					http.Logger(logger),
 					http.Context(ctx),
 					http.Config(cfg),
-					http.Metrics(metrics),
+					http.Metrics(m),
```

Comment:
What is wrong with the variable name?

Reply:
it collides with the package name and is discouraged

Reply:
then let us find something better, `m` alone is imho even worse.

Reply:
Why? The lifetime of this variable is 12 lines. You can get that down to 5 if you rearrange the code. So following `short lifetime` -> `short name` mantra, `m` seems a very good name for this var.

Reply:
Your explanation doesn’t convince me. You can, according to the guidelines, but what is the value? There is no added value in having a shorter variable name.

Reply:
Of course there is. 
- It is much easier to read. Simpling reading `m` takes less time than reading `metricsVarOnlyUsedOverAFewLines`
- You can deduct that if you see a variable like `m` or `r` the declaration must be just a few lines up, while with longer names you know that they do exist for longer. This can be especially be useful when overwriting or redeclaring such values.
- It's quicker to write.

On the other side I would rather argue that there is no added value by using a longer variable name.

Reply:
I appreciate your points.

We should always have readability as our highest goal. If we are aligned there, this is great.

> It is much easier to read. Simpling reading m takes less time than reading metricsVarOnlyUsedOverAFewLines

questionable. Could also mean "what was this `m` thingy? where is it initalized?"

> You can deduct that if you see a variable like m or r the declaration must be just a few lines up, while with longer names you know that they do exist for longer. This can be especially useful when overwriting or redeclaring such values.

point taken.

> It's quicker to write.

No argument, start typing, IDE writes the rest 😄

Reply:
we had this discussion already last time - in the go ecosystem such one char variables are quire common and we are all used to it. why fight against a commonly accepted practice?

let's use tab instead of 2 spaces then :dancers:

Reply:
> we had this discussion already last time - in the go ecosystem such one char variables are quire common and we are all used to it. why fight against a commonly accepted practice?
> 

Why just do stuff when it makes no sense? Having too short variables is IMHO a known "bad pattern" in the GoLang universe.

> let's use tab instead of 2 spaces then 👯

👯

Reply:
> Having too short variables is IMHO a known "bad pattern" in the GoLang universe.

no - https://go.dev/wiki/CodeReviewComments#variable-names

Reply:
> Why just do stuff when it makes no sense? Having too short variables is IMHO a known "bad pattern" in the GoLang universe.

I strongly disagree! 
In my opinion the java like "make your var names most explicit" method is a bad pattern in the golang universe.

Reply:
⏲️  Please stop. We have too much opinion here. Everybody can find arguments for the one ot the other direction.

I remember times where we had bad reva code where single char variables were used across more that several hundred lines and then re initialized in the middle.

@dragotin Made a strong point then, using his 30 years of experience with C++ We agreed that golang has a tendency to have "too short" variable names.

Let us not waste more time please, i agree in the current case it may make some sense.

But we should be very focused in each code review, to avoid that in the future.